docs/development/style-guide.md

```diff
@@ -21,6 +21,32 @@ Second sentence on a new line.
 And so on.
 ```
 
+## Avoid contractions/possessives
+
+Avoid:
+
+```markdown
+don't
+won't
+doesn't
+shouldn't
+wouldn't
+manager's
+file's
+```
```

Comment:
I've also seen `bot's` a multiple times, should I add this as well?

Reply:
I don't think it's confusing if people say "the bot's schedule" or something like that

Reply:
I agree with `@rarkins`. It's okay to use `the bot's x`. 😄